keras/applications/regnet.py

```diff
@@ -0,0 +1,1549 @@
+        "depths": [2, 6, 15, 2],
+        "widths": [96, 192, 432, 1008],
+        "group_width": 48,
+        "default_size": 224
+    },
+    "x040": {
+        "depths": [2, 5, 14, 2],
+        "widths": [80, 240, 560, 1360],
+        "group_width": 40,
+        "default_size": 224
+    },
+    "x064": {
+        "depths": [2, 4, 10, 1],
+        "widths": [168, 392, 784, 1624],
+        "group_width": 56,
+        "default_size": 224
+    },
+    "x080": {
+        "depths": [2, 5, 15, 1],
+        "widths": [80, 240, 720, 1920],
+        "group_width": 120,
+        "default_size": 224
+    },
+    "x120": {
+        "depths": [2, 5, 11, 1],
+        "widths": [224, 448, 896, 2240],
+        "group_width": 112,
+        "default_size": 224
+    },
+    "x160": {
+        "depths": [2, 6, 13, 1],
+        "widths": [256, 512, 896, 2048],
+        "group_width": 128,
+        "default_size": 224
+    },
+    "x320": {
+        "depths": [2, 7, 13, 1],
+        "widths": [336, 672, 1344, 2520],
+        "group_width": 168,
+        "default_size": 224
+    },
+    "y002": {
+        "depths": [1, 1, 4, 7],
+        "widths": [24, 56, 152, 368],
+        "group_width": 8,
+        "default_size": 224
+    },
+    "y004": {
+        "depths": [1, 3, 6, 6],
+        "widths": [48, 104, 208, 440],
+        "group_width": 8,
+        "default_size": 224
+    },
+    "y006": {
+        "depths": [1, 3, 7, 4],
+        "widths": [48, 112, 256, 608],
+        "group_width": 16,
+        "default_size": 224
+    },
+    "y008": {
+        "depths": [1, 3, 8, 2],
+        "widths": [64, 128, 320, 768],
+        "group_width": 16,
+        "default_size": 224
+    },
+    "y016": {
+        "depths": [2, 6, 17, 2],
+        "widths": [48, 120, 336, 888],
+        "group_width": 24,
+        "default_size": 224
+    },
+    "y032": {
+        "depths": [2, 5, 13, 1],
+        "widths": [72, 216, 576, 1512],
+        "group_width": 24,
+        "default_size": 224
+    },
+    "y040": {
+        "depths": [2, 6, 12, 2],
+        "widths": [128, 192, 512, 1088],
+        "group_width": 64,
+        "default_size": 224
+    },
+    "y064": {
+        "depths": [2, 7, 14, 2],
+        "widths": [144, 288, 576, 1296],
+        "group_width": 72,
+        "default_size": 224
+    },
+    "y080": {
+        "depths": [2, 4, 10, 1],
+        "widths": [168, 448, 896, 2016],
+        "group_width": 56,
+        "default_size": 224
+    },
+    "y120": {
+        "depths": [2, 5, 11, 1],
+        "widths": [224, 448, 896, 2240],
+        "group_width": 112,
+        "default_size": 224
+    },
+    "y160": {
+        "depths": [2, 4, 11, 1],
+        "widths": [224, 448, 1232, 3024],
+        "group_width": 112,
+        "default_size": 224
+    },
+    "y320": {
+        "depths": [2, 5, 12, 1],
+        "widths": [232, 696, 1392, 3712],
+        "group_width": 232,
+        "default_size": 224
+    }
+}
+
+BASE_DOCSTRING = """Instantiates Regnet architecture.
```

Comment:
You need to actually use this. See https://github.com/keras-team/keras/blob/6c719b1a2229d554dd7714c22ff7a5a8a84e0ae7/keras/applications/efficientnet_v2.py#L1261 as an example.